assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -50,7 +50,7 @@ function requestData(paymentFieldName: PaymentField,
   if (state.user.firstName !== null && state.user.firstName !== undefined
     && state.user.lastName !== null && state.user.lastName !== undefined
     && state.user.email !== null && state.user.email !== undefined) {
-    const monthlyContribFields: MonthlyContribFields = {
+    const regularContribFields: MonthlyContribFields = {
```

Comment:
Shouldn't this type change to RegularContribFields?

Reply:
Yes! thanks for spotting that.